server/src/helpers/errors.ts

```diff
@@ -90,14 +96,36 @@ const errors: Record<number, E> = {
     code: ApiError.INVALID_ACCOUNT_ADDRESS,
     message: 'Account address is invalid',
     retriable: false,
-  },
+  }
 };
 
-export function throwError(errorCode: ApiError, details?: object) {
+export function constructRosettaError(errorCode: ApiError, details?: object) {
   const { message, retriable, code } = errors[errorCode];
-  throw Error.constructFromObject({ code, message, retriable, details });
+  return Error.constructFromObject({ code, message, retriable, details });
+}
+
+export function throwError(errorCode: ApiError, details?: object) {
+  throw constructRosettaError(errorCode, details);
 }
 
 export const allErrors = Object.values(errors).map(
   ({ code, message, retriable }) => new Error(code, message, retriable),
 );
+
+export function isRosettaError(error: object): boolean {
```

Comment:
Added this as a way to check that a given object adheres to the required fields in the interface. Let me know if there's a better way to do this. :)